modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceProductMedia.testcase

```diff
@@ -112,12 +112,49 @@ definition {
 	}
 
 	@description = "This is a test for COMMERCE-6026. RemoveAnImage"
-	@ignore = "true"
 	@priority = "3"
 	test RemoveAnImage {
 		property portal.acceptance = "false";
 
-		// TODO COMMERCE-6026 RemoveAnImage
+		CommerceNavigator.gotoPortlet(
+			category = "Product Management",
+			portlet = "Catalogs");
+
+		Click(locator1 = "Button#PLUS");
+
+		CommerceEntry.addNewCatalogsEntry(
+			catalogCurrency = "US Dollar",
+			catalogLanguage = "English (United States)",
+			catalogName = "Master");
+
+		AssertClickNoError(
+			locator1 = "Button#SUBMIT",
+			value1 = "Submit");
+
+		Refresh();
```

Comment:
Please remove the creation for the Master catalog because is already created when we do the reindex into the setUp (lines 15-20)